ansible/roles_ocp_workloads/ocp4_workload_example/readme.adoc

```diff
@@ -167,7 +128,7 @@ platform: rhpds
 cloud_provider: none
 
 # --- Config
-env_type: ocp-workloads
+env_type: ocp4-cluster
```

Comment:
Why this change? You use the config `ocp-workloads` to deploy workloads onto an existing cluster.

Reply:
Oops.. duh.  Reverting.